templates/vue/utils/hydra.ts

```diff
@@ -0,0 +1,21 @@
+import { get, has, mapValues } from "lodash";
+
+interface HydraData {
+  "{{hydraPrefix}}member": HydraData[];
+}
+
+export function normalize(data: HydraData) {
+  if (has(data, "{{hydraPrefix}}member")) {
+    // Normalize items in collections
+    data["{{hydraPrefix}}member"] = data["{{hydraPrefix}}member"].map((item) => normalize(item));
+
+    return data;
+  }
+
+  // Flatten nested documents
```

Comment:
Since nested resources are flattened, I don't think the embedded code is working.
Could you test it?

Reply:
I checked and this file is not used at all. It must be legacy. Do you think it's important or can I delete it ?

Reply:
You can delete it then.